spk/hassio/Makefile

```diff
@@ -0,0 +1,45 @@
+SPK_NAME = hassio
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = $(shell git rev-parse --short HEAD)
```

Comment:
Revision must be incremental number. Please start at 1

Reply:
This is to create a "reproducablee build" ie tracking which build that is running.

I'll change it to 2019-224-1 to make every one happy. 🎅🏿

Reply:
@fredrike Why not simply go with 

```
SPK_VERS = 1.0.0
SPK_REV = 1
```

and increment with every new build and PR from here on? I agree this should not be date dependent and if there is a numbering schema in place it should follow that. Seems like a very minor fix that could be applied and unblock this entire PR.